bundles/extension.rst

```diff
@@ -129,6 +129,11 @@ read more about it, see the ":doc:`/bundles/configuration`" article.
 Adding Classes to Compile
 -------------------------
 
+.. versionadded:: 3.3
+    This technique is discouraged and the ``addClassesToCompile()`` has been
```

Comment:
I think we should reword this a bit as when #7193 is merged, we will also explain the `addAnnotatedClassesToCompile()` method in this section.

Reply:
I don't know what to do here. What were you thinking exactly? Thanks!